dNavigation/dTerrain/RawHeightMap.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <cstdint>
+#include <vector>
+
+#include "BinaryIO.h"
+
+class RawHeightMap {
+public:
+	RawHeightMap();
+	RawHeightMap(std::ifstream& stream, float height, float width);
+	~RawHeightMap();
+
+	uint32_t m_Unknown1;
+	uint32_t m_Unknown2;
+	uint32_t m_Unknown3;
+	uint32_t m_Unknown4;
```

Comment:
Why are these unknown?

Reply:
It is a custom format, we don't know what those values are.